monitor/main.py

```diff
@@ -220,6 +224,63 @@ def offline_logger(self, validator, steps):
         with open(self.report_dir / filename, "w") as f:
             json.dump({"validator": encode_hex(validator), "missed_steps": steps}, f)
 
+    def equivocation_logger(self, equivocated_block_hashes):
+        """Log a reported equivocation event.
+
+        Each new equivocation report is logged into a new file, using a counter
+        as unique identifier.
+        Logged information are the proposer of the blocks, the height at which
+        all blocks have been equivocated and a list of all block hashes with
+        their timestamp.
+        Additionally two representing blocks are logged with their RLP encoded
+        header and related signature, which can be used for an equivocation
+        proof on reporting a validator.
+        """
+
+        # Use the first two blocks as representational data for the equivocation proof.
+        block_hash_one = equivocated_block_hashes[0]
+        block_hash_two = equivocated_block_hashes[1]
+
+        block_one = get_canonicalized_block(self.w3.eth.getBlock(block_hash_one))
+        block_two = get_canonicalized_block(self.w3.eth.getBlock(block_hash_two))
+
+        # All blocks share the same proposer and height, so just pick the first.
```

Comment:
`assert`s are stronger than comments:
```suggestion
        assert block_one.proposer == block_two.proposer and block_one.height == block_two.height
```

Reply:
Okay, I thought about it. The point is, that actually the meaning is that **all** blocks have the same proposer and height. To `assert` this, I would need to query all blocks and assert them. Do you think using the first two representing blocks is enough?